### DISCUSSION TITLE
flooded tuner

### DISCUSSION THREAD

**Discussion by AutomGuy** [merbanan/rtl_433#2555] [Q&A]
Hi at all,

at my home i have many enviroment sensors based on 433,92 Mhz. At a specific day rtl_433 decodes nothing. I have two system with two different sdr sticks. these phenomenon is at both sticks. software was not changed. i could not belive that both sticks at the same time going into to the dark. i take a pc with rtl_433 on it , take a sensor and the original sdr stick as mentioned above and drive an half km away. start the rtl_433 software and it happend what i have expected. -> The sdr stick works.
Now my question: How can i found out with rtl_433 wich signal floods my sdr tuner at home? and where is these source of interference located.  

Thx in advance for hints or ideas

**Comment by zuckschwerdt** [comment]
You can watch the noise levels with e.g. `rtl_433 -Y autolevel -M level -M noise` -- if there is jamming you should see a suddenly much higher noise.

You can also stop rtl_433 and write a file for 2 minutes, e.g. `rtl_433 -w myfile_433.92M_250k.cu8 -T 120` then drop on https://triq.org/pdv/ to inspect.

Or run with rtl_tcp pass-through `rtl_433 -F rtl_tcp` then use any rtl_tcp capable UI and watch the live waterfall.

**Comment by AutomGuy** [comment]
thx for the fast reply. i will start with your recommendation

**Comment by AutomGuy** [comment]
Hi Christian,

i have started with `rtl_433 -Y autolevel -M level -M noise`
and this is the result. but a can not interpret that
```

Current signal level 2.7 dB, estimated noise -8.7 dB                                                                                                                                                    
Current signal level 2.7 dB, estimated noise -3.6 dB                                                                                                                                                    
Current signal level 2.7 dB, estimated noise -0.7 dB                                                                                                                                                    
Current noise level 2.7 dB, estimated noise 2.3 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.6 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB                                                                                                                                                      
Current noise level 2.7 dB, estimated noise 2.7 dB 
```

**Reply by zuckschwerdt** [reply]
You would want to see noise of around -15 to -30. Above -15 is not really enough room for a signal, above -9 will stop working, and the >0 you see is cursed.

**Comment by AutomGuy** [comment]
And this is the output from [ https://triq.org/pdv/]( https://triq.org/pdv/)
![pic](https://github.com/merbanan/rtl_433/assets/83695859/ad2565ee-bab6-4600-8cbf-e7c280ce0304)

**Reply by zuckschwerdt** [reply]
Yeah, that's a solid carrier with harmonics -- the band is unusable for you. Try the SDR on a portable computer, fix the gain and wander around to locate the source, maybe. Might also be worth it to switch off everything in your house and pull the breakers, just to be sure it's not a bad device in your house.

**Comment by AutomGuy** [comment]
its the gray at the right side the carrier or the pink one?

**Reply by zuckschwerdt** [reply]
Those are just the distribution of levels. The solid yellow/white horizontal line is the one -- it should not be there, it should be an empty blue/green field.

**Reply by AutomGuy** [reply]
thx christian for explanation.

**Reply by AutomGuy** [reply]
I also have my neighbors under suspicion. he has a RV with all sorts of inverters and charge controllers(from china). Maybe these are also send out harmonics. The RV is parked fairly close to my property

**Comment by AutomGuy** [comment]
yeah may be i have a device with almost empty batteries. i have read in other forums that 433Mhz devices in that case going to send a carrier. i try to remove all batteries from my devices and put new ones in. and then i hope the problem will be gone.

**Reply by AutomGuy** [reply]
Hello Christian,
I found the culprit. It was a temperature sensor with a low battery. He sent this carrier. everything runs normal now.
Many thx for your support with noise level and the waterfall diagram.

**Reply by zuckschwerdt** [reply]
That's great to hear!
